overview/bitstreams.md

```diff
@@ -0,0 +1,17 @@
+# DSpace Command-Line Interface (CLI)
+## Bitstream Management
+
+### dspace-cli
+| File | Procedure | Description | 
+| --- | --- | --- |
+| `thesis_central/set_default_bitstream_read.rb` | `doit` | Set the access policy for Mudd senior thesis items |
+
+### dspace-jruby
+| Class | Procedure | Description | 
+| --- | --- | --- |
+| `DBitstream` | `.all` | retrieve all bitstreams |
+| `DBitstream` | `.find` | find a bitstream using an ID |
+
+### tcsh operations
```

Comment:
Should I be bringing in the tcsh scripts from https://github.com/PrincetonUniversityLibrary/dspace-operations and commenting upon these directly? Many of these seem to be clean, and they seem to constitute a layer of CLI tools.